runtime-modules/working-group/src/lib.rs

```diff
@@ -1192,17 +1193,18 @@ decl_module! {
 
             ensure!(amount > Zero::zero(), Error::<T, I>::ZeroTokensFunding);
             ensure!(
-                Balances::<T>::can_slash(&account_id, amount),
+                Balances::<T>::usable_balance(&account_id) >= amount,
                 Error::<T, I>::InsufficientTokensForFunding
             );
 
             //
             // == MUTATION SAFE ==
             //
 
-            Self::set_working_group_budget(wg_budget.saturating_add(amount));
+            // An account is allowed to die when funding working group budget
+            burn_from_usable::<T>(&account_id, amount)?;
 
-            let _ = Balances::<T>::slash(&account_id, amount);
+            Self::set_working_group_budget(wg_budget.saturating_add(amount));
```

Comment:
Same point here about budget interaction.

Reply:
Fixed in https://github.com/Joystream/joystream/pull/4164/commits/deaf7f35d2f97f1068c51df5364962d755b043b2